src/ci/docker/run.sh

```diff
@@ -33,7 +33,11 @@ ci_dir="`dirname $script_dir`"
 src_dir="`dirname $ci_dir`"
 root_dir="`dirname $src_dir`"
 
-objdir=$root_dir/obj
+if isCI; then
```

Comment:
We need to add `source "$ci_dir/shared.sh"` above this, otherwise `isCI` is undefined when you run the `./src/ci/docker/run.sh` script locally.

Reply:
Oops. Done.